File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/IssueList/IssueListTeams.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IssueListTeams.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "IssueListForm"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDBEdit"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDropDownEdit"
#pragma link "cxEdit"
#pragma link "cxLookupEdit"
#pragma link "cxMaskEdit"
#pragma link "cxTextEdit"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDBLookupEdit"
#pragma link "cxGraphics"
#pragma link "cxLabel"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxNavigator"
#pragma resource "*.dfm"
TfrmTeams *frmTeams;
//---------------------------------------------------------------------------
__fastcall TfrmTeams::TfrmTeams(TComponent* Owner)
  : TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/IssueList/IssueListTeams.pas
unit IssueListTeams;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, IssueListForm, ImgList, ComCtrls, ToolWin, ExtCtrls, StdCtrls,
  cxMaskEdit, cxDropDownEdit, cxLookupEdit, cxDBLookupComboBox, cxControls,
  cxContainer, cxEdit, cxTextEdit, cxDBEdit, DBCtrls, cxDBLookupEdit,
  cxNavigator, cxGraphics, cxLookAndFeels, cxLookAndFeelPainters, cxLabel;

type
  TfrmTeams = class(TfrmBasic)
    Label9: TcxLabel;
    cxDBTextEdit1: TcxDBTextEdit;
    Label5: TcxLabel;
    cxDBLookupComboBox3: TcxDBLookupComboBox;
    Label1: TcxLabel;
    cxDBLookupComboBox1: TcxDBLookupComboBox;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmTeams: TfrmTeams;

implementation

{$R *.dfm}

uses
  IssueListForms, IssueListConst, IssueListData;

initialization

  FormManager.RegisterForm(TeamsFormID, TfrmTeams, 'PROJECT TEAMS');

end.
 
